scrapyjs/request.py

```diff
@@ -38,21 +42,48 @@ def __init__(self,
             splash_meta['splash_url'] = splash_url
         if splash_headers is not None:
             splash_meta['splash_headers'] = splash_headers
+        if dont_process_response:
+            splash_meta['dont_process_response'] = True
+        else:
+            splash_meta.setdefault('magic_response', magic_response)
+        if dont_send_headers:
+            splash_meta['dont_send_headers'] = True
+
+        if session_id is not None:
```

Comment:
I'm not sure here, but I think I'd find it less confusing if default value for ``session_id`` would be ``None``, and it would be passed to ``splash_meta`` regardless of the endpoint.
Hm, now I see why is that - right now ``session_id`` is useless for other endpoints, because ``render.png`` does not currently return cookies, right?